eqcorrscan/__init__.py

```diff
@@ -11,6 +11,14 @@
 import sys
 import warnings
 
+from eqcorrscan.core.match_filter import (  # NOQA
+    match_filter, Detection, Party, Tribe, Family, Template)
+from eqcorrscan.core.subspace import Detector, read_detector  # NOQA
+from eqcorrscan.core.lag_calc import lag_calc  # NOQA
+
+from eqcorrscan.utils.correlate import (  # NOQA
```

Comment:
worth adding the registration function here as well? It makes sense if we want users to be able to get to it quickly.

Reply:
:+1: Done.